pytest_rts/utils/common.py

```diff
@@ -162,3 +162,15 @@ def calculate_func_lines(src_code) -> Dict[str, Tuple[int, int]]:
     parsed_src_code = ast.parse(src_code)
     func_lines = function_lines(parsed_src_code, len(src_code.splitlines()))
     return {x[0]: (x[1], x[2]) for x in func_lines}
+
+
+def filter_and_sort_pytest_items(test_set, pytest_items, runtimes) -> List[Item]:
+    """Selected pytest items based on found tests
+    ordered by their runtimes
+    """
+    selected = list(filter(lambda item: item.nodeid in test_set, pytest_items))
+    updated_runtimes = {
+        item.nodeid: runtimes[item.nodeid] if item.nodeid in runtimes else sys.maxsize
```

Comment:
Is `if item.nodeid in runtimes` enough to filter out Nones?

```pytest
  runtimes = {"a": None}
  assert "a" in runtimes, True
```

I suppose if value is None `sorted()` will fail as well. Unit test for `filter_and_sort_pytest_items()` is missing, maybe all the options could be tested there?

Reply:
Yes the bug was actually caused by a situation exactly like that but I thought an extra check was not required because the root cause was fixed. Should it still be added?

Reply:
I would add it and also a couple of unit tests